configuration/addons.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: documentation
+---
+
+{% include base.html %}
+
+# Installation of Add-ons
+
+Depending on the [package]({{base}}/configuration/packages.html) you have choosen during your first time setup, there are already some pre-installed add-ons.
```

Comment:
`{{base}}` doesn't exist in VuePress (the site base is in `$site.base` but it's unnecessary to add it, it's handled by the framework), so this would normally fail, however to maintain compatibility with Jenkins it's transformed into `/docs/` by the prepare-docs.rb build script.

So we could leave this and let the prepare-docs script replace it, but I believe we decided to drop that Jekyll compatibility altogether, especially since it's a new page?
In that case it should be replaced everywhere it appears:
`{{base}}/configuration/packages.html` -> `/docs/configuration/packages.html`

Reply:
Thanks for clarifying this!
We should start a collection in team discussions, with things that change while we are moving to vuepress.